src/DynamoCoreWpf/ViewModels/Watch3D/IWatch3DViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+using Autodesk.DesignScript.Interfaces;
+using HelixToolkit.Wpf.SharpDX;
+
+
+namespace Dynamo.Wpf.ViewModels.Watch3D
```

Comment:
The interfaces folder is most likely going away, BTW, when we do the namespace reorganization next sprint. I agree that we need to document every one of our public contracts. See http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8392.